Blish HUD/GameServices/ContentService.cs

```diff
@@ -132,7 +132,7 @@ public enum FontStyle {
 
         private static Texture2D TextureFromFileSystem(string filepath) {
             if (!File.Exists(REF_FILE)) {
-                Console.WriteLine($"{REF_FILE} is missing!  Lots of assets will be missing!");
+                Logger.Warn("{refFileName} is missing!  Lots of assets will be missing!", REF_FILE);
```

Comment:
The usage of `refFileName` is confusing. Either stick to `{}` with the format string format, or use the string interpolation `$"{REF_FILE}"` please.

Reply:
Could you please elaborate on that first option?  NLog best practice is to allow for the additional params to be sent to the target logger.  https://github.com/NLog/NLog/wiki/How-to-use-structured-logging

Reply:
Ah, didn't know of that! I didn't use NLog in a *long* time, so I missed the structured logging update. I have to read into that.